### DISCUSSION TITLE
v0.5.1-beta.1

### DISCUSSION THREAD

**Discussion by sheikhhaziq** [sheikhhaziq/gyawun_music#25] [General]
## What's Changed
* Added support for Right to left direction
* Added Chinese translation file by @thekocanlhk in https://github.com/sheikhhaziq/vibemusic/pull/19
* Added Arabic translation file by @thekocanlhk in https://github.com/sheikhhaziq/vibemusic/pull/20
* Added support to save songs to favorites.
* Added Support to change location.
* Minor bugs fixed

**Full Changelog**: https://github.com/sheikhhaziq/vibemusic/compare/0.4.2...0.5.1-beta.1

<hr /><em>This discussion was created from the release <a href='https://github.com/sheikhhaziq/vibemusic/releases/tag/v0.5.1-beta.1'>v0.5.1-beta.1</a>.</em>

**Comment by senolfndk** [comment]
can you add theme options and file share

**Reply by sheikhhaziq** [reply]
What theme option fo you want. Can you describe it a bit.
And also share song option will require other person to have this app.

**Comment by senolfndk** [comment]
black blue orange brown yellow green themes

**Comment by senolfndk** [comment]
for example share feature like this
![Screenshot_20230104-200031_YouTube Music](https://user-images.githubusercontent.com/52818836/210610705-9ef06605-242e-46a8-a4c9-4a393c7571c4.jpg)
![Screenshot_20230104-200347](https://user-images.githubusercontent.com/52818836/210610713-a654a74a-740b-4566-9d46-8d53e922196e.jpg)
